### ISSUE TITLE
Glances crash after installing module for shown GPU informations on Windows 10

### ISSUE THREAD

**Issue by brodokk** [nicolargo/glances#1800]
**Describe the bug**
After installing the module for Nvidia GPU monitoring Glances crash.

**To Reproduce**
1. Install a fresh installation of python 3.9.1 (64bit) on Windows 10
2. Install Glances via pip with some other modules like `windows-curses`, `sparklines`, `netifaces` , `wifi`, `pySMART`, `pymdstat`, `py-cpuinfo` (Glances is ok at this point)
2. Install the module `py3nvml`
3. Start Glances without any option (`python -m glances`)
3. See error

**Expected behavior**
The GPU stats are shown without error in Glances.

**Desktop (please complete the following information):**
 - Windows 10 (19090)
 - Glances v3.1.6 with PsUtil v5.8.0

**Logs**
````
2021-01-24 12:50:40,039 -- INFO -- Start Glances 3.1.6
2021-01-24 12:50:40,040 -- INFO -- CPython 3.9.1 and psutil 5.8.0 detected
2021-01-24 12:50:40,042 -- INFO -- Search glances.conf file in C:\Users\fuck_off_microsoft\AppData\Roaming\glances\glances.conf
2021-01-24 12:50:40,042 -- INFO -- Search glances.conf file in C:\Users\fuck_off_microsoft\AppData\Roaming\glances\glances.conf
2021-01-24 12:50:40,043 -- WARNING -- PsUtil can not grab processes gids ('Process' object has no attribute 'gids')
2021-01-24 12:50:40,049 -- INFO -- Start GlancesStandalone mode
2021-01-24 12:50:40,107 -- WARNING -- Error loading Docker Python Lib. Docker plugin is disabled (No module named 'docker')
2021-01-24 12:50:40,108 -- ERROR -- docker plugin - Can not connect to Docker (name 'docker' is not defined)
2021-01-24 12:50:40,449 -- WARNING -- Wifi lib is not compliant with Python 3, Wifi plugin is disabled
````

**Glances output**
````
Traceback (most recent call last):
  File "C:\Users\fuck_off_microsoft\AppData\Local\Programs\Python\Python39\lib\runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Users\fuck_off_microsoft\AppData\Local\Programs\Python\Python39\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "C:\Users\fuck_off_microsoft\AppData\Local\Programs\Python\Python39\lib\site-packages\glances\__main__.py", line 29, in <module>
    glances.main()
  File "C:\Users\fuck_off_microsoft\AppData\Local\Programs\Python\Python39\lib\site-packages\glances\__init__.py", line 145, in main
    start(config=config, args=args)
  File "C:\Users\fuck_off_microsoft\AppData\Local\Programs\Python\Python39\lib\site-packages\glances\__init__.py", line 114, in start
    mode.serve_forever()
  File "C:\Users\fuck_off_microsoft\AppData\Local\Programs\Python\Python39\lib\site-packages\glances\standalone.py", line 154, in serve_forever
    loop = self.__serve_forever()
  File "C:\Users\fuck_off_microsoft\AppData\Local\Programs\Python\Python39\lib\site-packages\glances\standalone.py", line 124, in __serve_forever
    self.stats.update()
  File "C:\Users\fuck_off_microsoft\AppData\Local\Programs\Python\Python39\lib\site-packages\glances\stats.py", line 238, in update                                                                                                              
    self._plugins[p].update_views()
  File "C:\Users\fuck_off_microsoft\AppData\Local\Programs\Python\Python39\Lib\site-packages\glances\plugins\glances_gpu.py", line 107, in update_views
    super(Plugin, self).update_views()
  File "C:\Users\fuck_off_microsoft\AppData\Local\Programs\Python\Python39\lib\site-packages\glances\plugins\glances_plugin.py", line 501, in update_views
    '_zero': self.views[i[self.get_key()]][key]['_zero'] if i[self.get_key()] in self.views and key in self.views[i[self.get_key()]] else True}
KeyError: '_zero'
````

**Comment by nicolargo** [comment]
It looks like a bug introducing in the last commit before the release of Glances 3.1.6.

I will have a look as soon as possible.

Sorry for the bug.

**Comment by nicolargo** [comment]
Should be corrected in the patch version 3.1.6**.1**, please update Glances with pip.

Can you tell me back if it solve the issue ?

**Comment by brodokk** [comment]
No problem and thank you for your speed fix!

All work fine I will close this issue.
